k8s/prod/common/bsp/bulk-scan-orchestrator.yaml

```diff
@@ -24,6 +24,7 @@ spec:
         IDAM_API_URL: "https://idam-api.platform.hmcts.net"
         TRANSFORMATION_URL_DIVORCE: ""
         TRANSFORMATION_URL_FINREM: ""
+        UPDATE_URL_PROBATE: ""
```

Comment:
Do you think we need this?

Reply:
if we want to override , if we do not want to use in prod we need.

Reply:
Don't think so we need this as it is derived from value yaml

Reply:
we do not want to use this in prod?

Reply:
[discussion was moved to dev channel]